deployment/update_production.sh

```diff
@@ -44,8 +44,8 @@ sudo -H -u "$USERNAME" "$ENVDIR/bin/python" manage.py dumpdata --natural-foreign
 sudo -H -u "$USERNAME" "$ENVDIR/bin/pip" install -r requirements.txt
 # sometimes, this fails for some random i18n test translation files.
 sudo -H -u "$USERNAME" "$ENVDIR/bin/python" manage.py compilemessages || true
-sudo -H -u "$USERNAME" "$ENVDIR/bin/python" manage.py scss --production
-sudo -H -u "$USERNAME" "$ENVDIR/bin/python" manage.py ts compile --fresh
+sudo -H -u "$USERNAME" "$ENVDIR/bin/python" manage.py scss --production || true
+sudo -H -u "$USERNAME" "$ENVDIR/bin/python" manage.py ts compile --fresh || true
```

Comment:
These lines always fail. Note, that even before this PR:
> Could not find sass command
and 
> Could not find npx command
Now, the returncode is actually not zero and more prominent error messages are shown

Reply:
I think I'd rather have npm installed here. @niklasmohrin what do you think?

Reply:
Yeah, everything `*production` should fail eagerly